Docs.html

```
@@ -1,1503 +1,1320 @@
+        <div class="example-container">
+          <div class="example-title">Example</div>
+          <div class="example-code">&lt;dyn id="loadButton" ajax="data.json" insert="dynResult"&gt;Load Data&lt;/dyn&gt; &lt;div id="dynResult"&gt;&lt;/div&gt;</div>
+          <div class="example-output">
+            <dotp>
+              <div id="loadButton" class="button">Load Data</div>
+              <div id="dynResult" style="margin-top: 10px; padding: 10px; border: 1px solid #ddd;">Data will be loaded here</div>
+            </dotp>
+          </div>
+        </div>
+      </div>
+      <div class="attribute-section">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </div> ], start tag match failed [ <div class="attribute-section"> ] on line 645.

(tag-pair)